app/src/main/AndroidManifest.xml

```diff
@@ -23,7 +23,11 @@
         <activity android:name=".RegisterActivity" />
         <activity android:name=".TeacherLoginActivity" />
         <activity android:name=".StudentLoginActivity" />
-
+        <activity android:name=".TeacherDashboard.ClassHomeActivity" />
+        <activity android:name=".LoginSelectionActivity.TeacherDashboard.TeacherDashboardActivity" />
```

Comment:
I think this is an incorrect activity path and is causing errors when trying to log in as a teacher. I believe it should not have the .LoginSelectionActivity on the front of it.